### PR TITLE
Py tests tidss updates v2

### PR DESCRIPTION
**Pull request by jsarha**
Version 2 series.

### CODE REVIEW

File: py/tests/alpha-test.py [hunk 1]
```diff
@@ -2,50 +2,68 @@
 
 import pykms
 import time
+import argparse
+
+parser = argparse.ArgumentParser(description='Simple alpha blendig test.')
```
Comment (tomba) [inline py/tests/alpha-test.py]: "blendig"

File: py/tests/ctm_test.py [hunk 2]
```diff
@@ -22,44 +23,66 @@ def ctm_to_blob(ctm, card):
...
-mode = conn.get_default_mode()
+format = pykms.PixelFormat.ARGB8888
+if args.modename == None:
+    mode = conn.get_default_mode()
+else:
+    mode = conn.get_mode(args.modename)
+modeb = mode.to_blob(card)
 
-fb = pykms.DumbFramebuffer(card, mode.hdisplay, mode.vdisplay, "XR24");
+fb = pykms.DumbFramebuffer(card, mode.hdisplay, mode.vdisplay, "AR24");
```
Comment (tomba) [inline py/tests/ctm_test.py]: Why AR24?

File: py/tests/global_alpha_test.py [hunk 3]
```diff
@@ -2,12 +2,22 @@
 
 import pykms
 import time
-
+import argparse
+
+parser = argparse.ArgumentParser(description='Simple alpha blendig test.')
```
Comment (tomba) [inline py/tests/global_alpha_test.py]: "blendig", and not alpha test but global alpha.

File: py/tests/ctm_test.py (hunk 2 above)
Comment (jsarha) [reply py/tests/ctm_test.py]: Oh that is an obsolete test. Back when I implemented CTM for tidss the driver did not support XR24, and I did not want to do a context switch for that. I'll revert that back.

**Pull request closed without merging**